docs/API.md

```diff
@@ -142,9 +142,210 @@ Later, the subsequent logins can be done via `token`:
   },
 };
 
-{ response: { id: "3", users: [...]}
+{ response: { id: "3", users: [...]} }
 ```
 
+## Contact add
+
+```
+{
+  request: {
+    contact_add: {
+      first_name: "Name",
+      last_name: "Surname",
+      company: "UserCompany",
+      email: [ { type: work, value: "..." }, { type: home, value: "..." }, ...],
+      phone: [ { type: work, value: "..." }, { type: home, value: "..." }, ...],
+    },
+    id: "4",
+   },
+};
+
+{
+  response: {
+    id: "4",
+    contact: {
+      first_name: "Name",
+      last_name: "Surname",
+      company: "UserCompany",
+      user_id: "uId1",
+      email: [
+        { type: work, value: "...", matched_user_id: "uId5" },
+        { type: home, value: "..." },
+        ...
+        ],
+      phone: [
+        { type: work, value: "..." },
+        { type: home, value: "...", matched_user_id: "uId7" },
+        ...
+        ],
+      updated_at: "",
+      created_at: "",
+    }
+  }
+}
+```
+
+`email` or `phone` param is required, as well as `first_name` or `last_name`.
+
+## Contact batch add
+
+```
+{
+  request: {
+    contacts: [
+      {
+        first_name: "Name",
+        last_name: "Surname",
+        company: "UserCompany",
+        email: [ { type: work, value: "..." } ],
+        phone: [ { type: home, value: "..." } ],
+      },
+      {
+        first_name: "Name2",
+        last_name: "Surname2",
+        company: "UserCompany2",
+        email: [ { type: work, value: "..." } ],
+        phone: [ { type: home, value: "..." } ],
+      },
+      ...
+    ],
+    id: "5",
+   },
+};
+
+{
+  response: {
+    id: "5",
+    contacts: [
+      {
+        first_name: "Name",
+        last_name: "Surname",
+        company: "UserCompany",
+        user_id: "uId1",
```

Comment:
`user_id` is hidden field.

We also need `_id` field, which will be used in contact update request to identify which contact to update